src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -58,31 +59,24 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
         $dispatcher = $this->getEventDispatcher();
 
+        $options = array();
         if ($event = $input->getArgument('event')) {
             if (!$dispatcher->hasListeners($event)) {
-                $formatter = $this->getHelperSet()->get('formatter');
-
-                $formattedBlock = $formatter->formatBlock(
-                    sprintf('[NOTE] The event "%s" does not have any registered listeners.', $event),
-                    'fg=yellow',
-                    true
-                );
-
-                $output->writeln($formattedBlock);
+                $output->error(sprintf('The event "%s" does not have any registered listeners.', $event));
```

Comment:
In the screenshot, it was a warning. Is it really intended to be an error here ?